backend/src/Installer.test.ts

```diff
@@ -21,13 +22,17 @@ describe('backend/Installer', () => {
 
     beforeAll(() => {
       installer = new Installer({
-        name,
-        version,
+        resources: [
+          { name: 'remoteit', version: version.cli, url: version.url_cli },
+          { name: 'muxer', version: version.muxer, url: version.url_base },
+          { name: 'demuxer', version: version.demuxer, url: version.url_base },
+          { name: 'connectd', version: version.connectd, url: version.url_connectd },
+        ],
```

Comment:
Not sure it matters, but why duplicate the resources that are in the constants here?